### ISSUE TITLE
Pink screen and stalling during playback

### ISSUE THREAD

**Issue by ghost** [MapStory/mapstory-meta#413]
http://mapstory.org/maps/360 - this map flashed the pink screen every few years... doesn't stall playback

http://mapstory.org/data/geonode:Clean1 - this layer fails to play most times, and on occasion throws the pink screen

**Comment by bartvde** [comment]
the second url hangs my browser (Safari or Chrome)

**Comment by bartvde** [comment]
If I am logged in, the first one does not give my pink tiles.

**Comment by ischneider** [comment]
@elasher I tested the first one and saw no issues. sporadic pink tiles probably means the server was under heavy load, though it's hard to say.

The second one was configured to have 1000 year interval and also hung my browser. I reconfigured it to a year interval and the map played but obviously needs styling. This raises the (known and now ticketed) issue that we need a way to configure the time dimension after upload.

see #417
